### ISSUE TITLE
Error during notes conversion

### ISSUE THREAD

**Issue by dkuester** [BastiTee/tolino-notes#3]
When I try to convert my notes.txt to markdown I get this error:

```
Traceback (most recent call last):
  File "/usr/local/bin/tolino-notes", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/tolino_notes/__main__.py", line 58, in main
    opt_note = TolinoNote.from_unparsed_content(raw)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/tolino_notes/tolino_note.py", line 157, in from_unparsed_content
    page = int(re.sub(r'\s', ' ', prefix).split(' ')[-1])
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ValueError: invalid literal for int() with base 10: 'erleben"'
```

This was the command I used:
`tolino-notes --input-file /Users/dkuester/Downloads/notes.txt --output-dir /Volumes/Daten/Tolino --format md`

Here's the notes.txt that caused the issue:
[notes.txt](https://github.com/BastiTee/tolino-notes/files/12783615/notes.txt)

Thank you for your hellp.

**Comment by BastiTee** [comment]
Hi @dkuester,
thanks for the detailed description. I’ll be able to look into it soon, but with all the data available I am optimistic 👍

**Comment by BastiTee** [comment]
Hi @dkuester,
I just released 0.3.1 on pypi and [here](https://github.com/BastiTee/tolino-notes/releases/tag/0.3.1).
It fixes your issue. There has been yet another strange note format, that I haven't encountered before 🤷
